### PR TITLE
Audit article - INFODEV-2918: General Package Installation Guidelines

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c593cfa91c/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c593cfa91c/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-64eab46fc4/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b598ee3ce4/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e4eea64079/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-f537cf843e/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a84ddfa9a5/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3ef9520fe6/how-to/). :bow:

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by RackerKate** [comment]
@stephamon Please review.

**Comment by stephamon** [comment]
@RackerKate on it!

File: content/cloud-servers/general-package-installation-guidelines.md [hunk 1]
```diff
@@ -1,115 +1,153 @@
...
-#### APT/Dpkg (Debian, Ubuntu)
+When you run a command such as `yum install iptables`, the package manager
+(in this case, `Yum`) downloads the relevant package (`iptables`) from its own
+repositories and installs it on the system.
 
--   Install package:
+We recommend that you read the relevant documentation to learn more
+about each of these package managers.
+
+#### Additional information
```
Comment (stephamon) [inline content/cloud-servers/general-package-installation-guidelines.md]: I'd call this section "Linux package managers" and make it a h3 (###) level

File: content/cloud-servers/general-package-installation-guidelines.md [hunk 2]
```diff
@@ -1,115 +1,153 @@
...
+
+Every distribution that Cloud Servers offers comes with its own package
+manager. This article describes common package managers and provides example
```
Comment (stephamon) [inline content/cloud-servers/general-package-installation-guidelines.md]: The following sections describe common package managers and provide example commands...

File: content/cloud-servers/general-package-installation-guidelines.md [hunk 3]
```diff
@@ -1,115 +1,153 @@
...
+commands for different distributions.
+
+**Warning**: Avoid performing an in-place upgrade of any of the following
+package management systems or of all installed packages between versions on
+Rackspace Cloud Servers. This operation can break the configurations that
+enable Rackspace Cloud Servers to communicate with the hypervisors.
+
+#### APT (Debian and Ubuntu)
+
+Debian&trade; uses the dpkg packaging system. APT (Advanced Package Tool) is a
```
Comment (stephamon) [inline content/cloud-servers/general-package-installation-guidelines.md]: I'd rearrange to something more like this:

Advanced Package Tool (APT) is a command-line tool for interacting with dpkg, the packaging system used by Debian&trade;.

File: content/cloud-servers/general-package-installation-guidelines.md [hunk 4]
```diff
@@ -1,115 +1,153 @@
...
+Every distribution that Cloud Servers offers comes with its own package
+manager. This article describes common package managers and provides example
+commands for different distributions.
+
+**Warning**: Avoid performing an in-place upgrade of any of the following
+package management systems or of all installed packages between versions on
+Rackspace Cloud Servers. This operation can break the configurations that
+enable Rackspace Cloud Servers to communicate with the hypervisors.
+
+#### APT (Debian and Ubuntu)
```
Comment (stephamon) [inline content/cloud-servers/general-package-installation-guidelines.md]: Ubuntu should have a &reg; after it

File: content/cloud-servers/general-package-installation-guidelines.md [hunk 5]
```diff
@@ -121,122 +159,128 @@ and the [APT HOWTO](http://www.debian.org/doc/manuals/apt-howto/ "http://www.deb
...
--   Install package:
+Pacman is the package management tool for the Arch Linux&trade; distribution.
+Use the following commands with Pacman:
+
+-   Install a package:
 
         pacman -S PACKAGENAME
 
--   Remove package:
+-   Removea package:
```
Comment (stephamon) [inline content/cloud-servers/general-package-installation-guidelines.md]: "Remove a package"

Also, does this command keep the configuration files like in the previous sections?

File: content/cloud-servers/general-package-installation-guidelines.md (hunk 4 above)
Comment (RackerKate) [reply content/cloud-servers/general-package-installation-guidelines.md]: In the header? The one for Debian is in the regular text below it. Should I move both to the header?

Comment (stephamon) [reply content/cloud-servers/general-package-installation-guidelines.md]: Hm, I see your point there. I guess we can move it to the next mention of Ubuntu, which I think is in the next section.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0629a611f3/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0629a611f3/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-772b0e92e8/how-to/). :bow:

**Comment by RackerKate** [comment]
@stephamon Please review.

**Comment by stephamon** [comment]
@RackerKate I'm on it!

**Pull request merged**